policyfile.go

```diff
@@ -159,6 +162,15 @@ type NodeAttrGrantApp struct {
 	Domains    []string `json:"domains,omitempty" hujson:"Domains,omitempty"`
 }
 
+// ACLAttrConfig represents configuration for a custom device attribute.
+type ACLAttrConfig struct {
+	Type string `json:"type,omitempty" hujson:"Type,omitempty"`
+	// AllowSetByNode indicates if nodes can set this attribute via LocalAPI.
+	AllowSetByNode bool `json:"allowSetByNode,omitempty" hujson:"AllowSetByNode,omitempty"`
+	// BroadcastToPeers lists which nodes should receive this attribute value.
```

Comment:
nit: AFAIU this field will accept most/all values that are valid for the `dst` field in grants? If that's correct, maybe worth including the word destinations in this description to help clarify what kinds of values are valid.

```suggestion
	// BroadcastToPeers is a list of destinations which should receive this attribute value.
	// e.g. ["tag:admin"]
```